lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -953,13 +953,16 @@ def __init__(
         on_relation = self._charm.on[self._relation_name]
         self.framework.observe(
             on_relation.relation_created,
-            self._set_endpoint_on_relation_change,
+            self._on_relation_change,
```

Comment:
Any particular reason to adopt a less suggestive method name ?

Reply:
This is a Prometheus Lib, not a Grafana agent one.